packages/libs/client-utils/src/integration-tests/coin.int.test.ts

```diff
@@ -137,7 +138,7 @@ describe('createAccount', () => {
         account: accountTwo.account,
         keyset: {
           keys: [accountTwo.publicKey],
-          pred: 'keys-all',
+          pred: Predicate.keysAll,
```

Comment:
I prefer to see "keys-all". it still have code completion

Reply:
why would you prefer a hardcoded string over a variable?

Reply:
its not string it a literal (constant) type with only three valid options. 
it turns to string because of the error in the TPredicate
And if something is never going to change, then why should we put it inside a variable?